meson.build

```diff
@@ -0,0 +1,231 @@
+project('gst-shark', 'c',
+  version : '0.7.5.1',
+  meson_version : '>= 0.58',)
```

Comment:
Why is this specific version needed?

Reply:
Various reasons, for example if I use version 0.4, it gives me these warning:

WARNING: Project specifies a minimum meson_version '>= 0.4' but uses features which were added in newer versions:
 * 0.43.0: {'compiler.get_supported_arguments'}
 * 0.45.0: {'yield arg in option'}
 * 0.46.0: {'both_libraries'}
 * 0.51.0: {'dep.get_variable'}
 * 0.58.0: {'format strings', 'meson.global_build_root'}
Hence v0.58.0